source/languages/en/dataplatform/installing.md

```diff
@@ -61,26 +62,28 @@ If you are using or plan to use Spark, you must install Java 8 on **each node th
 
 If you are on Ubuntu, run the following to install Java 8:
 
-```Ubuntu
-$ sudo apt-get install software-properties-common
-$ sudo apt-get install python-software-properties
-$ sudo add-apt-repository ppa:webupd8team/java
-$ sudo apt-get update
-$ sudo apt-get install -y oracle-java8-installer
+```bash
+sudo apt-get install -y software-properties-common python-software-properties debconf-utils
```

Comment:
Leading $'s were requested to be removed for consistency, especially with existing Riak KV documentation, see http://docs.basho.com/riak/latest/ops/tuning/open-files-limit/